omnibus/resources/agent/msi/cal/targetver.h

```diff
@@ -5,7 +5,7 @@
 #endif
 
 #ifndef _WIN32_WINNT            // Specifies that the minimum required platform is Windows XP.
-#define _WIN32_WINNT 0x0501     // Change this to the appropriate value to target other versions of Windows.
+#define _WIN32_WINNT  _WIN32_WINNT_VISTA     // Change this to the appropriate value to target other versions of Windows.
```

Comment:
I guess the comment above should be updated to mention `Windows Vista` instead of XP.

Also, any reason `WINVER` is still set to XP? (i.e. `0x0501`, based on https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=vs-2017)

Reply:
windows builders are still on go 1.10

Reply:
WINVER was set to XP by default; I generally set it as low as I can and still get API requirements that I need (i.e. the "upgrade" to vista above was necessary for one of the APIs)